scipy/special/add_newdocs.py

```diff
@@ -6,8 +6,9 @@
 # _generate_pyx.py to generate the docstrings for the ufuncs in
 # scipy.special at the C level when the ufuncs are created at compile
 # time.
+from typing import Dict
 
-docdict = {}
+docdict: Dict[str, str] = {}
```

Comment:
This and below are instances where mypy can't infer the type because the container isn't initialized with anything.